setup.py

```diff
@@ -64,7 +64,7 @@ def run_tests(self):
     raise Exception('this library does not support python >2 and <3.5')
 elif sys.version_info[:2] >= (3, 5):
     install_requires.extend([
-        'aiohttp',
+        'aiohttp>=2.0.0',
```

Comment:
Even though aiohttp uses an [attempt at being humorous](http://sentimentalversioning.org/) for a versioning scheme, the scheme they uses still attempts to encode information.  I think.

Can we use `>=2.0.0,<3`?  Also, maybe add something for async_timeout.